packages/internal/src/babel/api.ts

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs'
```

Comment:
I think this should go into `build/babel/api`

Reply:
Makes sense